deploy/helm/templates/swo-agent-statefulset.yaml

```diff
@@ -0,0 +1,39 @@
+{{- if .Values.swoagent.enabled }}
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: {{ .Values.cluster.name | replace " " "-" }}-swo-agent
```

Comment:
Kubernetes naming convention has following rules (see https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-label-names)
* contain at most 63 characters
* contain only lowercase alphanumeric characters or '-'
* start with an alphabetic character
* end with an alphanumeric character

If you want to use name input by the users you would need to cover all these

Reply:
I still don't like this. It's not consistent. And can we trust that the cluster name is valid for this usage if we just replace a space with a dash? I'd rather fix the issue with the UAMS agent itself.

Reply:
I added some simple logic to use cluster name if fits k8s naming convention, if not it will use release name with suffix. It is not ideal but could work in this beta before we get proper validation of name in wizard.